Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -614,8 +656,8 @@ BoardInit (
   case PostSiliconInit:
     // To prevent from generating MCA for CLFLUSH flash region
     AsmMsrAnd32 (IA32_MC4_CTL, (UINT32)~BIT4);
-    // Enable GFX PCI command register if splash screen is required.
-    if (PcdGetBool (PcdSplashEnabled)) {
+    // Enable GFX PCI command register if framebuffer init is required.
+    if (PcdGetBool (PcdFramebufferInitEnabled)) {
```

Comment:
PcdGetBool (PcdIntelGfxEnabled) && PcdGetBool (PcdFramebufferInitEnabled)?

Reply:
I had thought about it.  The platform config itself should deal with the PCD dependency settings properly.   EX:  if Splash is enabled or FramebuferInit is enabled or Graphics console output is enabled,    then the IntelGfx must be enabled.  We cannot check all the dependencies everywhere.  I would like to centralize the check in platform config instead.

Reply:
Got it. looks good to me.